### PR TITLE
Previous / next navigation component

### PR DESCRIPTION
**Pull request by AccessPaul**
This branch builds on top of https://github.com/accessdigital/wilson/pull/30 is merged. Probably best reviewing and merging that one first. **Note: Getting Wilson features reviewed and merged is becoming more of a priority because we have a Wilson-based project with a very short deadline which is waiting for these updates before starting.**

- Introduces a "Previous / next navigation" paragraph which allows an editor to add cards to their page which link to the "previous" or "next" page or article.
- The editor picks the content via node reference fields and the templates automatically grab the title and image from the content (preference is for the teaser title & image fields, but will fallback to node title and article image where available).
- Paragraph contains the usual "settings" fields for layout, padding and animation etc, except for background image. I thought it best not to use this component with a background image for fear of the small cards being difficult to see. I could be persuaded otherwise though.
- This PR also corrects some issues with the Tailwind safelist, which reduces the overall CSS file size and restores some missing styles.

Demo showing the component added twice to the same page with different prev/next nodes and styling:

prev-next

Images dropped on mobile:

prev-next mobile

Desktop hover effect:

prev-next hover

CMS:

Screenshot 2022-09-17 at 22 15 49

### CODE REVIEW

File: themes/wilson_theme_starterkit/templates/content/node--next-card.html.twig [hunk 1]
```diff
@@ -0,0 +1,95 @@
+{#
+/**
+ * @file
+ * Theme override to display a teaser node.
```
Comment (rob-howells) [inline themes/wilson_theme_starterkit/templates/content/node--next-card.html.twig:4]: Comment could be updated here to reference next card node

File: themes/wilson_theme_starterkit/templates/content/node--next-card.html.twig [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    {% set title = node.field_teaser_title.0.value|check_markup('full_html') %}
+  {% endif %}
+
+  <div class="px-4 sm:px-6 py-3 flex items-center grow">
+    <p>
+      <span class="block text-xs text-grey-500 leading-relaxed">{{ 'Up next:'|t }}</span>
+      <span class="block text-sm md:text-base leading-snug font-medium">{{ title }}</span>
+    </p>
+  </div>
+</a>
```
Comment (rob-howells) [inline themes/wilson_theme_starterkit/templates/content/node--next-card.html.twig:95]: We should try and avoid using `<a>` tags to wrap big blocks of content - when a screen reader reaches this teaser it's going to read out information about the decorative image image as well as the 'up next: _title_' part, which is quite a lot of information to digest. 

As alternative, we could make only the 'up next: _title_' text (or perhaps even just the title text) a link and then use psudeo element on the link that is positioned over the entire teaser, which would give the whole teaser a clickable / focusable state whilst keeping the link text concise. And example of this approach can be found on the teasers on the BBC news homepage

File: themes/wilson_theme_starterkit/templates/content/node--previous-card.html.twig [hunk 3]
```diff
@@ -0,0 +1,95 @@
+{#
+/**
+ * @file
+ * Theme override to display a teaser node.
```
Comment (rob-howells) [inline themes/wilson_theme_starterkit/templates/content/node--previous-card.html.twig:4]: Comment could be updated here

File: themes/wilson_theme_starterkit/templates/content/node--previous-card.html.twig [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+  {% set title = node.label %}
+  {% if content.field_teaser_title|render %}
+    {% set title = node.field_teaser_title.0.value|check_markup('full_html') %}
+  {% endif %}
+
+  <div class="px-4 sm:px-6 py-3 flex items-center grow">
+    <p>
+      <span class="block text-xs text-grey-500 leading-relaxed">{{ 'Previous:'|t }}</span>
+      <span class="block text-sm md:text-base leading-snug font-medium">{{ title }}</span>
```
Comment (rob-howells) [inline themes/wilson_theme_starterkit/templates/content/node--previous-card.html.twig:92]: Same comment applies here regarding link text and decorative images

File: themes/wilson_theme_starterkit/templates/content/node--next-card.html.twig (hunk 1 above)
Comment (AccessPaul) [reply themes/wilson_theme_starterkit/templates/content/node--next-card.html.twig]: Good catch. I've updated.

File: themes/wilson_theme_starterkit/templates/content/node--previous-card.html.twig (hunk 3 above)
Comment (AccessPaul) [reply themes/wilson_theme_starterkit/templates/content/node--previous-card.html.twig]: Good catch. I've updated.

File: themes/wilson_theme_starterkit/templates/content/node--next-card.html.twig (hunk 2 above)
Comment (AccessPaul) [reply themes/wilson_theme_starterkit/templates/content/node--next-card.html.twig]: I like this idea and was easy to implement.

**Review by rob-howells — APPROVED** [review]

**Pull request merged**